src/OpenTelemetry.Instrumentation.Hangfire/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+﻿OpenTelemetry.Trace.HangfireInstrumentationOptions
```

Comment:
this package is going 1.0 stable right? Then this should be in the shipped.txt

Reply:
ok, fixing

Reply:
BTW, when will the 1.0 nuget release be available?

Reply:
In this comment @cijothomas, asked you to document semantic convention. It can be done in this repository or on the https://github.com/open-telemetry/opentelemetry-specification/tree/main/semantic_conventions/trace side.

Reply:
this spec (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/telemetry-stability.md#fixed-schema-telemetry-producers) is still experimental, but Hangfire would fall into this category. It can go stable, but it cannot change the shape of the telemetry. I'd suggest documenting the fields in the readme.md file. One example is https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.Runtime/README.md

Reply:
What about releasing it as another unstable beta version?

Maybe its too soon for a stable version, considering this discussion https://github.com/open-telemetry/opentelemetry-specification/issues/1347

Reply:
updated the changelog